django/urls/converters.py

```diff
@@ -0,0 +1,89 @@
+import uuid
+
+from django.utils import lru_cache
+
+
+class BaseConverter:
+    pass
+
+
+class IntConverter(BaseConverter):
+    regex = '[0-9]+'
+
+    def to_python(self, value):
+        return int(value)
+
+    def to_url(self, value):
+        return str(value)
+
+
+class StringConverter(BaseConverter):
+    regex = '[^/]+'
+
+    def to_python(self, value):
+        return value
+
+    def to_url(self, value):
+        return value
+
+
+class UUIDConverter(BaseConverter):
+    regex = '[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}'
+
+    def to_python(self, value):
+        return uuid.UUID(value)
+
+    def to_url(self, value):
+        return str(value)
+
+
+class SlugConverter(BaseConverter):
+    # Django has both `slug_re` and `slug_unicode_re`. In the case of URLs, the
```

Comment:
Maybe I missed a discussion, but `SlugField` also has an [allow_unicode](https://docs.djangoproject.com/en/dev/ref/models/fields/#django.db.models.SlugField.allow_unicode) option. Is the best solution to add a new converter for that use case? Or just use the string converter?